### PR TITLE
files updated

### PR DESCRIPTION
**Pull request by SumitSaundadkar**

### CODE REVIEW

**Review by TusharJade — COMMENTED** [review]
Good to go

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to me 👍

**Review by shristabaruah — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -4,22 +4,22 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Ecom-Web</title>
+    <title>Fashion-Mart</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
     <ul class="container">
         <li class="pages-list">
-           <a class="anchors" href="/HTML and CSS/home.html">Home Page</a>
+           <a class="anchors" href="/screens/produtlist/product.html">Home Page</a>
```
Comment (amarphule) [inline index.html]: I think the class name should be `anchor` as it's a single anchor tag.

File: screens/produtlist/product.css [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+      border-radius: 10px;
+      border: none;
+      padding: 5px;
+      text-align: center;
+      cursor: pointer;
+    
+      font-weight: 500;
+    }
+    .btn-icon-add {
+      background-color: gray;
```
Comment (amarphule) [inline screens/produtlist/product.css]: you could create a variable for color `gray` like `--gray-color`

File: screens/produtlist/product.css [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    }
+    
+    #filter-sidebar {
+      background-color: rgba(39, 39, 39, 0.898);
+      grid-area: sidebar;
+      margin-top: -2.6rem;
+      height: 100vh;
+      width: 100%;
+      text-align: center;
+      color: white;
```
Comment (amarphule) [inline screens/produtlist/product.css]: same as above

File: screens/produtlist/product.css [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+      grid-area: content2;
+    }
+    #content3 {
+      background-color: pink;
+      grid-area: content3;
+    }
+    /* footer {
+      background-color: aqua;
+      grid-area: footer;
+    } */
```
Comment (amarphule) [inline screens/produtlist/product.css]: remove comments

File: style.css [hunk 5]
```diff
@@ -19,6 +19,6 @@
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
-    background-color: rgb(202, 166, 226);
+    background-color: blueviolet;
```
Comment (amarphule) [inline style.css]: same as above

**Review by amarphule — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (SumitSaundadkar) [reply index.html]: thanks updated

File: screens/produtlist/product.css (hunk 2 above)
Comment (SumitSaundadkar) [reply screens/produtlist/product.css]: updated

**Pull request merged**